components/ithc/.terraform-version

```diff
@@ -1 +0,0 @@
-1.2.3
```

Comment:
why delete all the terraform-version files? if they are deleted they don't work locally? sometimes we symlink them to the root of the path but having the file in all components works too as renovate will update it